### PR TITLE
docs(suspensive.org): add infiniteQueryOptions doc

### PR DESCRIPTION
**Pull request by gwansikk**
# Overview

- removed the WIP label.

| ko | en |
| --- | --- |
| image|image|

## PR Checklist

- [x] I did below actions if need

1. I read the [Contributing Guide](https://github.com/toss/suspensive/blob/main/CONTRIBUTING.md)
2. I added documents and tests.

### CODE REVIEW

**Comment by gwansikk** [comment]
@kangju2000 Could you please take a look at Sandpack(example)?

**Comment by manudeli** [comment]
> @kangju2000 Could you please take a look at Sandpack(example)?

I checked it. It's cool!

**Review by manudeli — APPROVED** [review]
Let's merge this 🚀🚀🚀🚀🚀

File: .changeset/itchy-spies-vanish.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@suspensive/suspensive.org": patch
+---
+
+docs(suspensive.org): add infiniteQueryOptions doc
```
Comment (manudeli) [inline .changeset/itchy-spies-vanish.md:5]: Remove changeset please! suspensive.org don't have to be version bump

Comment (gwansikk) [reply .changeset/itchy-spies-vanish.md:5]: Oh, sorry, my mistake.

**Review by manudeli — APPROVED** [review]

**Pull request merged**